instances/technique/RNASequencing.jsonld

```diff
@@ -0,0 +1,17 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/RNASequencing",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": "A technique to determine the sequence of a RNA molecule.",
+  "description": null,
+  "interlexIdentifier": "http://uri.interlex.org/ilx_0782092",
+  "knowledgeSpaceLink": null,
+  "name": "RNA sequencing",
+  "preferredOntologyIdentifier": "http://purl.obolibrary.org/obo/OBI_0001177",
+  "synonym": [
+    "RNA sequencing assay",
```

Comment:
maybe also add the non abbreviated synonyms?

Reply:
done :)